macros/probing/dockable_probe.cfg

```diff
@@ -14,13 +14,11 @@ gcode:
     {% set function  = 'pre_' ~ params.FUNCTION %}
     {% set move  = params.MOVE|default(0) %}
     {% set speed  = params.MOVE_SPEED|default(0) %}
-    M400 # mandatory to save the new safe position
     RESTORE_GCODE_STATE NAME={function} MOVE={move} MOVE_SPEED={speed}
 
 [gcode_macro _ENTRY_POINT]
 gcode:
     {% set function  = 'pre_' ~ params.FUNCTION %}
-    M400 # mandatory to save the new safe position

```

Comment:
These two `M400` lines are really mandatory for slower Pi to be sure that the movement is finished and the buffer flushed before doing a save of the position.

FYI, these M400 lines in the official Klicky repo are also written by myself as I corrected a bug in the early klicky macros developpement, so that's why I know exactly why they are here :sweat_smile:

Reply:
I agree with that, but we need to save the position only in 2 cases:
- PROBE_CALIBRATE
- PROBE_ACCURACY

In all other case the previous position is not used.

If you look these 2 macros i have insert a M400 before calling ENTRY_POINT

Reply:
Ok for me then ! It doesn't change a lot of things afterall :)